test/framework/src/main/java/org/elasticsearch/test/rest/Stash.java

```diff
@@ -78,7 +86,10 @@ public boolean isStashedValue(Object key) {
      * The stash contains fields eventually extracted from previous responses that can be reused
      * as arguments for following requests (e.g. scroll_id)
      */
-    public Object unstashValue(String value) {
+    public Object unstashValue(String value) throws IOException {
+        if (value.startsWith("$body")) {
+            return response.evaluate(value.substring("$body".length()), this);
```

Comment:
This isn't strictly required but it seems super convenient. It helps a lot with the simple `s/ / /` syntax in the docs - no need to round trip through the stash explicitly.